File: Codeforces/Round640Div4/q3.cpp
#include<iostream>
#include <cstdio>
#include <math.h>
#include<vector>
#define ll long long
using namespace std;

int main()
{
  ll i, t, n, k, rem, ans;
  freopen("input.txt","r",stdin);
  cin>>t;
  for(i=0; i<t; i++){
    cin>>n>>k;
    rem = k%(n-1);
    if(rem==0){
      cout<<(n*(k/(n-1)) - 1)<<endl;
    }
    else{
      cout<<(n*(k/(n-1)) + rem)<<endl;
    }
  }
}

File: Codeforces/Round642Div3/q3.py
input1 = input()
input1 = int(input1)

for i in range(input1):
    n = int(input())
    ans = 0;
    blocks = 0;
    for j in range(n,1,-2):
        blocks = blocks + (4*(j-1));
        ans = ans + blocks
    print(ans)
